docs/utils_usage.md

```diff
@@ -374,7 +374,14 @@ the `OPENAI_API_KEY` and `SERPER_API_KEY` environment variables.
 ```bash
 task run:command -- extract_companies_from_image http://example.com/logo.png
 ```
+## Extract Primary and Secondary Color from a website
 
+`get_website_information.py` detects primary and secondary colors used in a company website. It requires
```

Comment:
nit: can we update to say `answers some basic questions about the website`, as we are doing more than color detection here?

Reply:
I have changed it in my PR, will push in a different PR.